src/plugins/notebook/components/Notebook.vue

```diff
@@ -431,7 +431,9 @@ export default {
         },
         async updateDefaultNotebook(notebookStorage) {
             const defaultNotebookObject = await this.getDefaultNotebookObject();
-            if (defaultNotebookObject.identifier.key !== notebookStorage.notebookMeta.identifier.key) {
+            if (!defaultNotebookObject) {
+                setDefaultNotebook(this.openmct, notebookStorage);
+            } else if (defaultNotebookObject.identifier.key !== notebookStorage.notebookMeta.identifier.key) {
```

Comment:
We should always be checking keystrings. Not including the namespace property will cause issues down the road

Reply:
or is it better to use, `objectEquals` from `object-utils.js`?

Reply:
using `makeKeyString `